frontend/src/components/pages/MockSearch/mockReviews.ts

```diff
@@ -139,16 +139,36 @@ const mockMultipleBookReview: Review = {
     firstName: "jessie",
     lastName: "peng",
   },
-  books: [hp1, hp2],
+  books: [hp1, hp2, hp1, hp2],
   updatedAt: 12,
   publishedAt: 20171111212,
   createdAt: 14,
 };
 
+const mockMultipleBookReviews2: Review = {
+  reviewId: 12,
+  body: "body of review",
+  byline: "byline",
+  featured: true,
+  createdByUser: {
+    firstName: "jessie",
+    lastName: "peng",
+  },
+  books: [mockGenericBookData, mockLongTitleBookData],
+  updatedAt: 12,
+  publishedAt: 20171111333,
+  createdAt: 14,
+};
+
 const mockReviews: Review[] = [
```

Comment:
wait why are we calling them all mock

Reply:
It's because we're just using this fake data to test what the frontend would look like. Eventually, we're gonna get rid of the mock data and just use data fetched from the database.